semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/messaging/MessagingSemanticConventionUtils.java

```diff
@@ -37,4 +46,43 @@ public static boolean isRabbitMqBackend(Event event) {
     return SpanAttributeUtils.containsAttributeKey(event, RawSpanConstants.getValue(RabbitMq.RABBIT_MQ_ROUTING_KEY))
         || SpanAttributeUtils.containsAttributeKey(event, OtelMessagingSemanticConventions.RABBITMQ_ROUTING_KEY.getValue());
   }
+
+  public static Optional<String> getKafkaBackendURI(Event event) {
+    if (!isKafkaBackend(event)) {
+      return Optional.empty();
+    }
+
+    if(SpanAttributeUtils.containsAttributeKey(event, MESSAGING_URL)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, MESSAGING_URL));
+    }
+    return DbSemanticConventionUtils.getBackendURIForOtelFormat(event);
+  }
+
+  public static boolean isKafkaBackend(Event event) {
+    if(SpanAttributeUtils.containsAttributeKey(event, MESSAGING_SYSTEM)) {
+      return KAFKA_SYSTEM_VALUE.equals(SpanAttributeUtils.getStringAttributeWithDefault(
+          event, MESSAGING_SYSTEM, StringUtils.EMPTY));
+    }
+    return false;
+  }
+
+  public static Optional<String> getSqsBackendURI(Event event) {
+    if (!isSqsBackend(event)) {
+      return Optional.empty();
+    }
+
+    if(SpanAttributeUtils.containsAttributeKey(event, MESSAGING_URL)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, MESSAGING_URL));
+    }
+
+    return DbSemanticConventionUtils.getBackendURIForOtelFormat(event);
+  }
+
+  public static boolean isSqsBackend(Event event) {
+    if(SpanAttributeUtils.containsAttributeKey(event, MESSAGING_SYSTEM)) {
+      return SQS_SYSTEM_VALUE.equals(SpanAttributeUtils.getStringAttributeWithDefault(
```

Comment:
it seems that for Aws SQS, messing.url should suffice - e.g https://github.com/zalando-incubator/opentracing-sqs-java

Reply:
From Otel Semantics, messaging.system must be set to the name of the system that we are using, hence added support for messaging.system. 

And don't think existence of messaging.url is a sufficient condition for sqs as that can be set for other messaging backends too.